ReactiveCocoa/Swift/Event.swift

```diff
@@ -49,7 +57,7 @@ public enum Event<T, E: ErrorType> {
 	public func map<U>(f: T -> U) -> Event<U, E> {
 		switch self {
 		case let .Next(value):
-			return Event<U, E>.Next(Box(f(value.unbox)))
+			return Event<U, E>.Next(Box(f(value.value)))
```

Comment:
Sorry for the misunderstanding. 😓 For iOS 7, we totally can't use dynamic framework. So `Result`'s source files also should be included in an app target, `internal` means within the app target for the case.